src/dom-manipulation.ts

```diff
@@ -7,9 +8,9 @@ import {
   convertMeterPerSecondToKilomentersPerHour,
 } from "./utils/utils.js";
 
-export function appendFlightInformationToFlightInfoContainer(flight) {
+export function appendFlightInformationToFlightInfoContainer(flight: IFlights) {
```

Comment:
Possible spelling error on IFlights?

Reply:
Looks good to me, where do you see the mistake so I can resolve it.